### PR TITLE
Collect only required MyRocks metrics

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: collector/engine_rocksdb.go [hunk 1]
```diff
@@ -22,6 +22,74 @@ const (
...
+
+		"rocksdb.no.file.opens":  newDesc(rocksDB, "no_file_opens", "rocksdb.no.file.opens"),
+		"rocksdb.no.file.closes": newDesc(rocksDB, "no_file_closes", "rocksdb.no.file.closes"),
+		"rocksdb.no.file.errors": newDesc(rocksDB, "no_file_errors", "rocksdb.no.file.errors"),
+
+		"rocksdb.write.self":    newDesc(rocksDB, "write_self", "rocksdb.write.self"),
+		"rocksdb.write.other":   newDesc(rocksDB, "write_other", "rocksdb.write.other"),
+		"rocksdb.write.timeout": newDesc(rocksDB, "write_timeout", "rocksdb.write.timeout"),
+		"rocksdb.write.wal":     newDesc(rocksDB, "write_wal", "rocksdb.write.wal"),
+	}
```
Comment (AlekSi) [inline collector/engine_rocksdb.go]: @arvenil please check for typos

Comment (arvenil) [reply collector/engine_rocksdb.go]: That's why I would generate those names rather than doing copy&paste ;)

Comment (arvenil) [reply collector/engine_rocksdb.go]: Looks good, any idea what `wat` stands for?

File: collector/engine_rocksdb.go [hunk 2]
```diff
@@ -37,17 +105,16 @@ func parseRocksDBStatistics(data string) ([]prometheus.Metric, error) {
 			continue
 		}
 
-		name := strings.Replace(m[1], ".", "_", -1)
```
Comment (arvenil) [inline collector/engine_rocksdb.go]: oh, yes like it was here :D

File: collector/engine_rocksdb.go (hunk 1 above)
Comment (AlekSi) [reply collector/engine_rocksdb.go]: You mean `wal`? Write-ahead log, most likely.

I use map there for three reasons:
* to have an explicit list of metrics we use, and skip unknown ones;
* to have a place for better metric descriptions.
* to be able to give alternative names to metrics, and to keep a metric name when RocksDB changes an internal name.

The downside is copy&paste for that iteration. :(

**Pull request merged**